### PR TITLE
Update some C# examples for 4.0

### PR DESCRIPTION
**Pull request by raulsntos**
- Rename members that have been renamed in Godot's C# API for 4.0.
- Change `delta` parameter type to `double`.
- Ensure parameters match base declaration.
- Other minor code fixes.
- Related PR in the Core repo: https://github.com/godotengine/godot/pull/72462.

I have renamed everything I could find that was using the old names but I likely missed some, I intentionally left out the getting started tutorial as well as some physics tutorials that seemed too outdated and needed bigger changes so I thought it'd be better to update those in a separate PR.

### CODE REVIEW

**Review by mhilbrunner — APPROVED** [review]
Spotted no issues, but will let it stay open a bit longer to give people more versed in Godot's C# stuff a chance to spot things too :) Nice work!

File: tutorials/inputs/input_examples.rst [hunk 1]
```diff
@@ -43,20 +43,20 @@ Examples:
...
     }
 
-    public override void _PhysicsProcess(float delta)
+    public override void _PhysicsProcess(double delta)
     {
         if (Input.IsActionPressed("move_right"))
         {
             // Move as long as the key/button is pressed.
-            position.x += speed * delta;
+            position.X += speed * delta;
```
Comment (paulloz) [inline tutorials/inputs/input_examples.rst:59]:
```suggestion
            position.X += speed * (float)delta;
```

File: tutorials/physics/ray-casting.rst [hunk 2]
```diff
@@ -114,11 +114,11 @@ may be used. For example:
...
  .. code-tab:: csharp
 
-    public override void _PhysicsProcess(float delta)
+    public override void _PhysicsProcess(double delta)
     {
-        var spaceState = GetWorld2d().DirectSpaceState;
+        var spaceState = GetWorld2D().DirectSpaceState;
         // use global coordinates, not local to node
-        var query = PhysicsRayQueryParameters2D.create(new Vector2(), new Vector2(50, 100));
+        var query = PhysicsRayQueryParameters2D.Create(new Vector2(0, 0), new Vector2(50, 100));
```
Comment (paulloz) [inline tutorials/physics/ray-casting.rst:121]:
```suggestion
        var query = PhysicsRayQueryParameters2D.Create(Vector2.Zero, new Vector2(50, 100));
```

File: tutorials/2d/custom_drawing_in_2d.rst [hunk 3]
```diff
@@ -490,7 +490,7 @@ smaller value, which directly depends on the rendering speed.
 
  .. code-tab:: csharp
 
-    public override void _Process(float delta)
+    public override void _Process(double delta)
     {
         _angleFrom += _rotationAngle * delta;
```
Comment (paulloz) [inline tutorials/2d/custom_drawing_in_2d.rst:495]:
```suggestion
        _angleFrom += _rotationAngle * (float)delta;
```

File: tutorials/2d/custom_drawing_in_2d.rst [hunk 4]
```diff
@@ -490,7 +490,7 @@ smaller value, which directly depends on the rendering speed.
...
         _angleTo += _rotationAngle * delta;
```
Comment (paulloz) [inline tutorials/2d/custom_drawing_in_2d.rst:496]:
```suggestion
        _angleTo += _rotationAngle * (float)delta;
```

**Review by paulloz — COMMENTED** [review]
Pesky float conversions 😭

**Comment by raulsntos** [comment]
Thank you so much for the reviews. I missed a bunch of those conversions :sweat_smile:

**Comment by mhilbrunner** [comment]
Thank you! 🎉 Merged.

**Pull request merged**